src/settings/components/folder-select-modal/folder-select-modal.tsx

```diff
@@ -0,0 +1,197 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+// this component is work in progress for when the mails sync is implemented
```

Comment:
Sync in mails is already implemented, what is missing?

Reply:
I think giuliano meant the folders sync, we are considering lifting the management of the user folders to the shell

Reply:
yep, I was referring to that exactly :)